src/V3EmitCImp.cpp

```diff
@@ -25,6 +25,7 @@
 #include "V3UniqueNames.h"
 
 #include <map>
+#include <memory>
```

Comment:
Don't think you need this now without shared_ptr

Reply:
Removed.